pkg/agent/baker_test.go

```diff
@@ -18,80 +19,161 @@ func generateTestData() bool {
 	return os.Getenv("GENERATE_TEST_DATA") == "true"
 }
 
-var _ = Describe("Assert generated customData and cseCmd", func() {
-	DescribeTable("Generated customData and CSE", func(folder, k8sVersion string, configUpdator func(*datamodel.NodeBootstrappingConfiguration)) {
-		cs := &datamodel.ContainerService{
-			Location: "southcentralus",
-			Type:     "Microsoft.ContainerService/ManagedClusters",
-			Properties: &datamodel.Properties{
-				OrchestratorProfile: &datamodel.OrchestratorProfile{
-					OrchestratorType:    datamodel.Kubernetes,
-					OrchestratorVersion: k8sVersion,
+func getFakeContainerService(k8sVersion string) *datamodel.ContainerService {
+	cs := &datamodel.ContainerService{
+		Location: "southcentralus",
+		Type:     "Microsoft.ContainerService/ManagedClusters",
+		Properties: &datamodel.Properties{
+			TelemetryProfile: &datamodel.TelemetryProfile{
```

Comment:
do we use this?

Reply:
This is indeed from our containerservice, I obtained this from an existing cluster.

Reply:
Remove this for not used